engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -2256,15 +2256,15 @@ protected void initHostName() {
 
   protected void initDefaultMetrics(MetricsRegistry metricsRegistry) {
     metricsRegistry.createMeter(Metrics.ACTIVTY_INSTANCE_START);
-    metricsRegistry.createMeter(Metrics.ACTIVTY_INSTANCE_END);
+    metricsRegistry.createDbMeter(Metrics.ACTIVTY_INSTANCE_END);
 
-    metricsRegistry.createMeter(Metrics.JOB_ACQUISITION_ATTEMPT);
-    metricsRegistry.createMeter(Metrics.JOB_ACQUIRED_SUCCESS);
-    metricsRegistry.createMeter(Metrics.JOB_ACQUIRED_FAILURE);
-    metricsRegistry.createMeter(Metrics.JOB_SUCCESSFUL);
-    metricsRegistry.createMeter(Metrics.JOB_FAILED);
-    metricsRegistry.createMeter(Metrics.JOB_LOCKED_EXCLUSIVE);
-    metricsRegistry.createMeter(Metrics.JOB_EXECUTION_REJECTED);
+    metricsRegistry.createDbMeter(Metrics.JOB_ACQUISITION_ATTEMPT);
+    metricsRegistry.createDbMeter(Metrics.JOB_ACQUIRED_SUCCESS);
+    metricsRegistry.createDbMeter(Metrics.JOB_ACQUIRED_FAILURE);
+    metricsRegistry.createDbMeter(Metrics.JOB_SUCCESSFUL);
+    metricsRegistry.createDbMeter(Metrics.JOB_FAILED);
+    metricsRegistry.createDbMeter(Metrics.JOB_LOCKED_EXCLUSIVE);
+    metricsRegistry.createDbMeter(Metrics.JOB_EXECUTION_REJECTED);
 
     metricsRegistry.createMeter(Metrics.ROOT_PROCESS_INSTANCE_START);
 
```

Comment:
Does EXECUTED_DECISION_ELEMENTS meter left on purpost with `createMeter` instead of `createDbMeter`?

Reply:
I guess what I am wondering is: Why did we implement the flow node instance metric but not the decision elements metric for telemetry?

Reply:
Follow-up: https://jira.camunda.com/browse/CAM-12471